content/sensu-go/5.19/api/prune.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Prune API"
+description: "The Sensu prune API provides HTTP access to create pruning commands to delete resources that do not appear in a given set of Sensu objects from a file, URL, or STDIN. This reference includes an example for creating a prune command for your Sensu instance. Read on for the full reference."
+version: "5.19"
+product: "Sensu Go"
+menu:
+  sensu-go-5.19:
+    parent: api
+---
+
+## The `/prune/v1alpha` API endpoint
+
+{{% notice important %}}
+**IMPORTANT**: The prune API is an alpha feature in release 5.19.0 and may include breaking changes.
+{{% /notice %}}
+
+**COMMERCIAL FEATURE**: Access sensuctl pruning in the packaged Sensu Go distribution. For more information, see [Get started with commercial features][1].
+
+### `/prune/v1alpha` (POST)
+
+The `/prune/v1alpha` API endpoint provides HTTP POST access to create a pruning command to delete resources that do not appear in a given set of Sensu objects from a file, URL, or STDIN.
```

Comment:
> in a given set of Sensu objects from a file, URL, or STDIN.

That might be a bit misleading in the context of the API, since users will have to provide the Sensu objects themselves to curl. WDYT?

Reply:
I agree -- I copied that from sensuctl and then forgot to update it to match the API. Thank you.